extensions/wikia/ArchiveWikiForum/ArchiveWikiForumHooks.class.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+class ArchiveWikiForumHooks {
+
+    /**
+     * Show the notification informing the user that wiki forums have been disabled
+     * @param Article $article
+     * @param $outputDone
+     * @param $useParserCache
+     * @return bool
+     */
+    public static function onArticleViewHeader( &$article, &$outputDone, &$useParserCache ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unused function parameters "$outputDone, $useParserCache". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1172)